evm/chains/sharding/mainchain_handler/vmc_utils.py

```diff
@@ -0,0 +1,212 @@
+import os
+
+import rlp
+
+from viper import compiler
+
+from eth_abi import (
+    decode_abi,
+    encode_abi,
+)
+
+from eth_keys import KeyAPI
```

Comment:
You can modify this to use `from eth_keys import keys`.  The `key` object will be an instance of `KeyAPI` that you can use directly.

Reply:
no problem